src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,15 +1,43 @@
 # TO-DO: complete the helpe function below to merge 2 sorted arrays
-def merge( arrA, arrB ):
-    elements = len( arrA ) + len( arrB )
+def merge(arrA, arrB):
+    elements = len(arrA) + len(arrB)
     merged_arr = [0] * elements
     # TO-DO
-    
+
     return merged_arr
 
 
 # TO-DO: implement the Merge Sort function below USING RECURSION
-def merge_sort( arr ):
+def merge_sort(arr):
     # TO-DO
+    if len(arr) > 1:
+        mid = len(arr)//2
+        left = arr[:mid]
+        right = arr[mid:]
+
+        merge_sort(left)
+        merge_sort(right)
+
+        i = j = y = 0
+
+        while i < len(left) and j < len(right):
+            if left[i] < right[j]:
+                arr[y] = left[i]
+                i += 1
+            else:
+                arr[y] = right[j]
+                j += 1
+            y += 1
+
```

Comment:
Cool this looks clean. You can name your variables with more specific names for better readability